pkg/cedar/core/threat.go

```diff
@@ -0,0 +1,72 @@
+package cedarcore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+
+	apithreat "github.com/cmsgov/easi-app/pkg/cedar/core/gen/client/threat"
+)
+
+// GetThreat makes a GET call to the /threat endpoint
+func (c *Client) GetThreat(ctx context.Context, cedarSystemID string) ([]*models.CedarThreat, error) {
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return []*models.CedarThreat{}, nil
+	}
+
+	cedarSystem, err := c.GetSystem(ctx, cedarSystemID)
```

Comment:
We don't need to fetch the `cedarSystem` in this method; this line and the error check immediately afterward can be removed. We can pass `cedarSystemID` to the `c.GetAuthorityToOperate()` call instead of `cedarSystem.VersionID`, that method will translate the (version-independent) `cedarSystemID` to a version-specific system ID and use it to look up the ATOs, which is all we need in this method to query the `/threat` endpoint.

Reply:
Fixed with https://github.com/CMSgov/easi-app/pull/1653/commits/ff48cbbc950a0c8d735d7a87f1c2cf0b2b54010f.